blog/_posts/2016-08-20-hello-from-team-Crackers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: hello from Team Crackers!!
+layout: post
+created_at: Sat August 20 2016
+permalink: blog/2016-08-20-hello-from-team-Crackers
+current: blog
+author: Nikita and Nishtha
+twitter: Team Crackers
+categories:
+- student-posts
+---
+
+# **Team Crackers**
+
+![Team Crackers](/img/blog/2016/teamCrackers1.jpg)    
+
+hello everyone!!
+this is team Crackers. We are nikita and nishtha from Gandhinagar,India.
+we study at DA-IICT. We are in same batch and we have a fine tuning due to which we decided to apply for rails girls summer of code.Earlier we used to team up for coding competitions held online as well as in our college. We got to know about rails girls summer of code from one of our seniors and from then we looked out for coaches, teamed up with them and applied for this. This is our first time in open source community and our project is WeCare. 
+
+
+#### **NISHTHA**
+I like to define myself as an hyperactive, happy go lucky, borderline geeky, your friendly neighbourhood foodie. Intrigued often with more than how a phone looks, I like to keep up with consumer electronics, and the strides and fads of developer communities. However, my knowledge and love for technology was not more than skin deep back when I started my undergraduate studies as B.Tech in ICT at DA-IICT. I like being a proactively communicative and contributing member of the communities that I am a part of. 
+Being in a closely knit community, surrounded by technology enthusiasts, and talented developers, these two years have given me immense exposure and driven me to expand my horizons in many dimensions. From sweating when asked to print the fibonacci series on the console, to have successfully written data based Android apps, and I like to see my journey as a good one. However, it has a long way to go. In fact, I am currently pursuing a nanodegree program on Udacity, supported by a scholarship from Google and Tata Trusts.
+
+
+#### **NIKITA**
+I am currenty studying in 3rd year(undergrad) for the course Information and Communication Technology. I started coding with basic C and HTML since my high school. Later I got to know about web development, attended various workshops and discussions which acted as an ignition to working with web applications. Then I started studying various online courses and making informal projects. Later last summer I had my internship on educational web applications where I met very great people who taught me advance techniques in this field which is quite helpful for the project. In my free time I go out and play football with my team or fiddle around arts and crafts. Also I like travelling and mountain hiking.
+
+![Team Crakers](/img/blog/2016/teamCrackers2.jpg)
```

Comment:
this image is being shown on its side (vertical, not horizontal). Is this intentional? If not, please rotate the image locally and save it — git should be able to see the change and let you stage and commit it.